### PR TITLE
created a patch function for changing review votes

### PR DESCRIPTION
**Pull request by JamesATeasdale**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -188,3 +188,85 @@ describe("POST /api/reviews/:review_id/comments should post a new comment when g
 	});
 });
 
+describe("PATCH /api/reviews/:review_id should change vote value", () => {
+	test("202 if the request was accepted and the vote of the review was incremented", () => {
```
Comment (HeatherEver) [inline __tests__/app.test.js]: for this i would just expect a normal 200 :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -188,3 +188,85 @@ describe("POST /api/reviews/:review_id/comments should post a new comment when g
...
+						"https://images.pexels.com/photos/5350049/pexels-photo-5350049.jpeg?w=700&h=700",
+					created_at: "2021-01-18T10:01:41.251Z",
+					votes: 7,
+				};
+				expect(body.review.votes).toEqual(7);
+				expect(body.review.votes > data.reviewData[2].votes).toBe(true);
+				expect(body.review).toEqual(expectedBody);
+			});
+	});
+	test("202 if the request was accepted and the vote of the review was incremented", () => {
```
Comment (HeatherEver) [inline __tests__/app.test.js]: you're not incrementing the vote here, edit this test description pls

File: __tests__/app.test.js [hunk 3]
```diff
@@ -188,3 +188,85 @@ describe("POST /api/reviews/:review_id/comments should post a new comment when g
...
+				expect(body).toEqual({ msg: "Not found" });
+			});
+	});
+	test("400 if the body is in an incorrect format", () => {
+		return request(app)
+			.patch("/api/reviews/4")
+			.send({
+				person: "Mitch",
+				msg: "Hello! just letting you know that we did find your cat, he is doing well and is eating us out of pocket. Please pick him up ASAP!",
+			})
```
Comment (HeatherEver) [inline __tests__/app.test.js:253]: love this comment

File: __tests__/app.test.js [hunk 4]
```diff
@@ -188,3 +188,85 @@ describe("POST /api/reviews/:review_id/comments should post a new comment when g
...
+			.patch("/api/reviews/1")
+			.send({
+				votes: "Bela likes this",
+			})
+			.expect(400)
+			.then(({ body }) => {
+				expect(body).toEqual({ msg: "Bad Request" });
+			});
+	});
+});
```
Comment (HeatherEver) [inline __tests__/app.test.js:279]: really lovely testing here , can you just add one for 400 `/api/reviews/heather` <-- i just want to be featured 🥺

**Review by HeatherEver — APPROVED** [review]
really nice work here! just one more test to add! ⭐

File: __tests__/app.test.js (hunk 4 above)
Comment (JamesATeasdale) [reply __tests__/app.test.js:279]: it is done!

**Pull request merged**